music_grabber.py

```diff
@@ -0,0 +1,80 @@
+import os
+from youtube_dl import YoutubeDL
+from urllib.parse import urlparse
+from urllib.request import Request
+from urllib.request import urlopen
+from PIL import Image
+import config
+
+DOWNLOAD_DIR = config.DOWNLOAD_DIR
+
+
+class MusicDownloader:
+
+    def download(self, input_url, filesize_limit):
+        ydl_opts = {
+                'format': 'bestaudio[protocol^=http]',
+                'outtmpl': DOWNLOAD_DIR + '%(extractor)s_%(id)s.%(ext)s',
+                'writethumbnail': True
+        }
+        ydl = YoutubeDL(ydl_opts)
+        info = ydl.extract_info(input_url, download=False)
+        webpage_url = info['webpage_url']
+        extractor = info['extractor']
```

Comment:
overall looks alright, but can you split your code into more methods? To make it easier to read? Like

```
        ydl = YoutubeDL(ydl_opts)
        info = ydl.extract_info(input_url, download=False)
        webpage_url = info['webpage_url']
        extractor = info['extractor']

        if videoIsNotASong(...):
            raise ValueError(webpage_url + " is not under Music category")

        audio_filesize = __getFileSize(...)

        if int(audio_filesize) < filesize_limit:
            return __getAllSongInfo()
        else:
            raise ValueError(input_url + " audio file size (" +
                             str(audio_filesize) + ") is greater than the " +
                             "limit: (" + str(filesize_limit) + ")")
```

Reply:
maybe, will try that later

Reply:
Check latest commit

Reply:
nice bro i'll check it today